src/Doctrine/Odm/Extension/ParameterExtension.php

```diff
@@ -72,4 +83,26 @@ public function applyToItem(Builder $aggregationBuilder, string $resourceClass,
     {
         $this->applyFilter($aggregationBuilder, $resourceClass, $operation, $context);
     }
+
+    /**
+     * @param $values array<string, string>
+     *
+     * @throws ContainerExceptionInterface
+     * @throws NotFoundExceptionInterface
+     */
+    public function getFilter(AbstractFilter|string|null $filterId, array $values = []): ?FilterInterface
+    {
+        if ($filterId instanceof AbstractFilter) {
+            $filterId->setManagerRegistry($this->managerRegistry);
+            $filterId->setProperties($values);
```

Comment:
@soyuka `$parameter->getExtraProperties()` returns something like `[“_api_values” => “true”]`. It doesn't work with the current code if you set it to properties. However `$values` contains `[“active” => “true”]` which works.

Reply:
This is a wrong assumption, as values are not representing the properties, this should be one of: 

- `$parameter->getExtraProperties()['_properties']`
- `[$parameter->getProperty()]`
- `[$parameter->getKey()]`

For example when you alias a filter to a property values will contain [key => value] although it'd filter on a given property.

Reply:
Ok, I understand but  I've tried all 3 solutions to pass them to `setProperties()` and nothing works. I think I need an array with the parameter key (active) in the index and the filter value (true) in value, because that's the only solution that works actually, and I see something like that also in the following code, which then feeds `$values` if we check usages
![image](https://github.com/user-attachments/assets/6e4f7810-5391-42ee-9d4e-d07e80a06f77)
![image](https://github.com/user-attachments/assets/1dadb401-5696-4a50-abfa-1e61415e035d)